test/ci/create_aws_vm.yml

```diff
@@ -0,0 +1,57 @@
+---
+- name: Prepare aws_instance tag
+  set_fact:
+    aws_instance_tags: |
+      {
+        "kubernetes.io/cluster/{{ aws_cluster_id }}": "true",
+      }
+
+- name: Create a VM
+  ec2:
+    region: "{{ aws_region }}"
+    key_name: "{{ aws_key }}"
+    instance_type: "{{ outer_item.aws_flavor }}"
+    image: "{{ outer_item.aws_image | default(aws_image) }}"
+    wait: yes
+    group: "{{ outer_item.aws_security_group }}"
+    count: 1
+    vpc_subnet_id: "{{ aws_subnet }}"
+    assign_public_ip: yes
+    instance_tags: "{{ aws_instance_tags |
+      combine({'Name': outer_item.name_template + '-' + item }) }}"
+  register: ec2
+  with_sequence: start=1 end={{ outer_item.amount }}
+
+- name: Attach the volume for gluster
```

Comment:
No need to do this.  You can just add a volume section to ec2 task above.:

```yaml
        volumes:
          - device_name: /dev/sda1
            volume_size: 100
            delete_on_termination: true
```

Reply:
This probably would look ugly as it needs to be conditional for Gluster nodes only

Reply:
Volumes takes a list, we can supply via inventory.

Reply:
Done, now controlled by `aws_volumes` for VM group